ui-modules/blueprint-composer/package.json

```diff
@@ -30,7 +30,8 @@
     "start": "node ./config/server.js",
     "test": "karma start ./config/test/karma.config.js",
     "lint": "./node_modules/.bin/eslint app/**/*.js",
-    "e2e": "protractor ./config/test/protractor.config.js"
+    "e2e": "protractor ./config/test/protractor.config.js",
+    "branding": "BROOKLYN_UI_BRAND_DIR=/Users/mykola/IdeaProjects/cloudpack-megan/package/branding npm run start"
```

Comment:
This looks like your home directory?

Reply:
Good catch, did not notice that. This is removed now.